test/cpp/end2end/async_end2end_test.cc

```diff
@@ -51,7 +51,6 @@
 
 using grpc::testing::EchoRequest;
 using grpc::testing::EchoResponse;
-using grpc::testing::kTlsCredentialsType;
```

Comment:
I don't think it was needed at all. It's not used anywhere in the file.

Reply:
Can we add it back in then since we don't need to make changes to this file at all? (It's best to keep the PR changes as narrow as possible.)

If we want to remove it, perhaps we can do so in a separate PR.